contracts/src/ethereum/L1Token.sol

```diff
@@ -4,7 +4,21 @@ pragma solidity ^0.8.0;
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
 contract L1Token is ERC20 {
-    constructor(uint256 amount) ERC20("L1Token", "L1T") {
+
+    ERC20 public REWARD_TOKEN;
+
+    constructor(uint256 amount, address rewardToken) ERC20("L1Token", "L1T") {
         _mint(msg.sender, amount);
+        REWARD_TOKEN = RewAAVE(rewardToken);
+    }
+
+    function claimRewardsToSelf(bool forceUpdate) external {
+      //noop
```

Comment:
This function is to be implemented in this PR or in another one?

Reply:
This function only exists to satisfy the signature for the tests.

Reply:
Ok, understood!